packaging/linux/arch/update_aur_packages.sh

```diff
@@ -41,20 +45,33 @@ sum_i386="$(sha256sum "$deb_i386" | awk '{print $1}')"
 deb_amd64="$(ls "$build_root"/deb/amd64/*.deb)"
 sum_amd64="$(sha256sum "$deb_amd64" | awk '{print $1}')"
 
+url_src_prefix="https://s3.amazonaws.com/prerelease.keybase.io/linux_binaries/deb/"
+src_prefix=$url_src_prefix
+if [ -n "${KEYBASE_LOCAL_BUILD:-}" ] ; then
+    src_prefix=""
+    cp $deb_i386 $keybase_bin_repo/keybase_${debver}_i386.deb
+    cp $deb_amd64 $keybase_bin_repo/keybase_${debver}_amd64.deb
+fi
+
 cat "$here/PKGBUILD.bin.in" \
   | sed "s/@@PKGVER@@/$pkgver/g" \
   | sed "s/@@SUM_i686@@/$sum_i386/g" \
   | sed "s/@@SUM_x86_64@@/$sum_amd64/g" \
+  | sed "s/@@SRC_PREFIX@@/$src_prefix/g" \
   > "$keybase_bin_repo/PKGBUILD"
 
 cat "$here/DOT_SRCINFO.bin.in" \
   | sed "s/@@PKGVER@@/$pkgver/g" \
   | sed "s/@@DEBVER@@/$debver/g" \
   | sed "s/@@SUM_i686@@/$sum_i386/g" \
   | sed "s/@@SUM_x86_64@@/$sum_amd64/g" \
+  | sed "s/@@SRC_PREFIX@@/$src_prefix/g" \
   > "$keybase_bin_repo/.SRCINFO"
 
-if git -C "$keybase_bin_repo" commit -am "version bump" ; then
+if [ -n "${KEYBASE_LOCAL_BUILD:-}" ] ; then
+    cd $keybase_bin_repo
+    makepkg
```

Comment:
As we have it here, this script does very different things based on whether `KEYBASE_LOCAL_BUILD` is set. Should we maybe factor out the top part, and move these final lines into the callers?